### PR TITLE
Plot RE nodes as open circles

### PR DESCRIPTION
**Pull request by hyanwong**
And add svg tests

### CODE REVIEW

File: tests/test_info.py [hunk 1]
```diff
@@ -186,3 +186,20 @@ def test_plots(self, fx_ti_2020_02_13, method):
...
+    def test_draw_pango_lineage_subtree(self, fx_ti_2020_02_13):
+        ti = fx_ti_2020_02_13
+        svg = ti.draw_pango_lineage_subtree("A")
+        svg2 = ti.draw_subtree(tracked_pango=["A"])
+        assert svg == svg2
+        assert svg.startswith("<svg")
+        for u in ti.pango_lineage_samples['A']:
+            assert f"node n{u}" in svg
+
+    def test_draw_subtree(self, fx_ti_2020_02_13):
```
Comment (jeromekelleher) [inline tests/test_info.py:199]: These don't do anything? Better to delete if they don't do any testing, will give a false sense of coverage otherwise.

**Review by jeromekelleher — APPROVED** [review]
Looks great! Just delete the two unused tests for now I think

File: tests/test_info.py (hunk 1 above)
Comment (hyanwong) [reply tests/test_info.py:199]: Do they not do anything? I intended them to work? Have I missed something?

Comment (hyanwong) [reply tests/test_info.py:199]: Oops, sorry: I must have missed a commit. Hang on.

Comment (hyanwong) [reply tests/test_info.py:199]: Apologies: my editor didn't save some changes before commit. Fixed now. These tests should work.

**Pull request merged**